app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class VotesController < ApplicationController
+  def create
+    service = CreateVoteService.new(get_votable, current_user, params)
+    service.call
+    redirect_to session[:back]
```

Comment:
I think you can just call `redirect_to :back` in this case, and not worry about setting this session variable in your views